### ISSUE TITLE
coefs fails with 2 column response in GLM

### ISSUE THREAD

**Issue by jslefche** [jslefche/piecewiseSEM#142]
```
library(piecewiseSEM)

# Create fake data
dat <- data.frame(
  successes = sample(1:5, replace = T),
  failures = sample(1:5, replace = T),
  x = runif(100)
)

dat$y <- dat$successes / with(dat, successes + failures)

# Create model 
mod1 <- glm(y ~ x, "binomial", dat)


coefs(mod1) # get std estimate

# Create model with cbind
mod2 <- glm(cbind(successes, failures) ~ x, "binomial", dat) 

coefs(mod2) # error
```

**Comment by padluq** [comment]
I tried this with the newest version and received the same error.

**Comment by jslefche** [comment]
Yes this is a known issue…and unfortunately one that has been around forever. Its on the to-do list but I’m not confident I will be able to implement it in the near term. Send me an email if you want to work it out together!

____________________________
Jonathan S. Lefcheck, Ph.D.
Tennenbaum Coordinating Scientist
MarineGEO: https://marinegeo.si.edu/
Smithsonian Institution
Phone: +1 (443) 482-2443
www.jonlefcheck.net<http://www.jonlefcheck.net>

**Comment by padluq** [comment]
Dr. Lefcheck:
Thanks for your response. I was able to get the model to run using an expanded version of the data, with each row representing a single bernoulli trail.

Since then, I've come upon another question, however.  In your summary.R file for the most recent version of piecewiseSEM, you wrote:
[cid:4368f35f-9b5c-4d71-a6db-66c61b1dd111]

In slide 5 of lecture 7 from your SEM course, you show an example using the argument "groups" within a summary call.  Is this argument currently functional?

I've been struggling to figure out how to handle a categorical grouping in my data.  As a glmm, I used a categorical fixed effect because there are only two levels (two years), but I haven't found the best way to handle it in psem.  Dr. Byrnes has an example in one of his lecture slidedecks, but it's not yet clear to me.  Are there any other tutorials or examples you know of that might add to my understanding?

Thanks again,
Adrian
________________________________

**Comment by jebyrnes** [comment]
FYI, I've updated my categorical lectures slides - see https://jebyrnes.github.io/semclass/schedule.html

If you're interested i an interaction with a category, in the latest version of the development brach, you'll get those coefficients out nicely. See also in the lecture slides the bowen et al example if you are not looking at an interaction.

Unfortunately, we don't have categorical standardized coefficients implemented yet - it's next on my list, but, will take a hair as I just landed back in the real world.

**Comment by andersonrm** [comment]
# Create model 
mod1 <- glm(y ~ x, "binomial", dat)

coefs(mod1) # get std estimate

# Create model with cbind
mod2 <- glm(cbind(successes, failures) ~ x, "binomial", dat) 

coefs(mod2) # error

I've come across this issue again today. In the interim, should users proceed with the fractional response approach and just ignore the non-integer warning? Theoretically, if N trials is large and P(success) is small, a workaround would be to model count(successes) ~ x, family ="poisson". However, these are conditions rarely satisfied.
